src/test/java/com/facebook/openwifirrm/optimizers/tpc/MeasurementBasedApApTPCTest.java

```diff
@@ -78,65 +77,86 @@ private static DeviceDataManager createDeviceDataManager() {
 		return deviceDataManager;
 	}
 
+	/**
+	 *
+	 * Creates a data model with 3 devices in only the given band.
+	 * All are at max_tx_power, which represents the first step in greedy TPC.
+	 *
+	 * @param band band (e.g., "2G")
+	 * @return a data model
```

Comment:
nit: omit `@return` when it doesn't add any info (for example, include it if the method can return null in some cases)